npm-shrinkwrap.json

```diff
@@ -1,322 +0,0 @@
-{
```

Comment:
Given there are no production dependencies to this project, the production shrinkwrap would produce an empty `shrinkwrap.json`.